src/api/Metadata.js

```diff
@@ -179,6 +189,17 @@ class Metadata extends File {
         return getProp(templates, 'data.entries', []);
     }
 
+    /**
+     * Gets metadata template schema by template key
+     *
+     * @param {string} templateKey - template key
+     * @return {Promise} Promise object of metadata template
+     */
+    getSchemaByTemplateKey(templateKey: string): Promise<{ data?: MetadataTemplate }> {
```

Comment:
Should this use the `MetadataTemplateSchemaResponse` type? If it's not used anywhere else, you can probably keep it inline and remove the named type, instead.